components/dashboard/src/projects/ProjectListItem.tsx

```diff
@@ -40,7 +41,14 @@ export const ProjectListItem: FunctionComponent<ProjectListItemProps> = ({ proje
                                 menuEntries={[
                                     {
                                         title: "New Workspace",
```

Comment:
**question(non-blocking)**: Could we measure how much this ~~button~~ link (New Workspace) is used on the projects page specifically? I created [a report](https://eu.mixpanel.com/s/bUrSY) (internal) but couldn't differentiate the clicks on the specific dropdown. Cc @jakobhero